<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>371</height>
   </rect>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles">
     <property name="title">
      <string>&amp;Recent Files</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/16x16/document-open-recent.png</normaloff>:/images/16x16/document-open-recent.png</iconset>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSaveAll"/>
    <addaction name="actionReload"/>
    <addaction name="actionOpenContainingFolder"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuCustomActions">
     <property name="title">
      <string>Cu&amp;stom Actions</string>
     </property>
     <addaction name="actionManageCustomActions"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionRename"/>
    <addaction name="actionFindReplace"/>
    <addaction name="separator"/>
    <addaction name="menuCustomActions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <widget class="QMenu" name="menuSampleSessions">
     <property name="title">
      <string>Open &amp;Sample</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/16x16/document-open.png</normaloff>:/images/16x16/document-open.png</iconset>
     </property>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionOnlineHelp"/>
    <addaction name="menuSampleSessions"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSession">
    <property name="title">
     <string>&amp;Session</string>
    </property>
    <widget class="QMenu" name="menuEvaluation">
     <property name="title">
      <string>&amp;Evaluation</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/16x16/view-refresh.png</normaloff>:/images/16x16/view-refresh.png</iconset>
     </property>
     <addaction name="actionEvalReset"/>
     <addaction name="actionEvalManual"/>
     <addaction name="actionEvalAuto"/>
     <addaction name="actionEvalSteady"/>
    </widget>
    <addaction name="menuEvaluation"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuTabSize">
     <property name="title">
      <string>Tab Size</string>
     </property>
    </widget>
    <addaction name="menuTabSize"/>
    <addaction name="actionIndentWithSpaces"/>
    <addaction name="actionShowWhiteSpace"/>
    <addaction name="actionLineWrapping"/>
    <addaction name="separator"/>
    <addaction name="actionSelectFont"/>
    <addaction name="actionDarkTheme"/>
    <addaction name="actionFullScreen"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSession"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBarMain">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionDelete"/>
   <addaction name="actionFindReplace"/>
   <addaction name="separator"/>
   <addaction name="actionEvalReset"/>
   <addaction name="actionEvalManual"/>
   <addaction name="actionEvalAuto"/>
   <addaction name="actionEvalSteady"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/document-open.png</normaloff>:/images/16x16/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/document-save.png</normaloff>:/images/16x16/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/application-exit.png</normaloff>:/images/16x16/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/edit-copy.png</normaloff>:/images/16x16/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/edit-paste.png</normaloff>:/images/16x16/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/help-about.png</normaloff>:/images/16x16/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About GPUpad</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/document-save-as.png</normaloff>:/images/16x16/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/document-new.png</normaloff>:/images/16x16/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/window-close.png</normaloff>:/images/16x16/window-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/edit-cut.png</normaloff>:/images/16x16/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/document-properties.png</normaloff>:/images/16x16/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Pre&amp;ferences...</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>C&amp;lose All</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/edit-delete.png</normaloff>:/images/16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="iconText">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="text">
    <string>Sa&amp;ve All</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/view-refresh.png</normaloff>:/images/16x16/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/edit-select-all.png</normaloff>:/images/16x16/edit-select-all.png</iconset>
   </property>
   <property name="text">
    <string>Select &amp;All</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/edit-undo.png</normaloff>:/images/16x16/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/edit-redo.png</normaloff>:/images/16x16/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionLineWrapping">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Line &amp;Wrapping</string>
   </property>
   <property name="toolTip">
    <string>Line Wrapping</string>
   </property>
  </action>
  <action name="actionShowWhiteSpace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show White&amp;space</string>
   </property>
   <property name="toolTip">
    <string>Show Whitespace</string>
   </property>
  </action>
  <action name="actionIndentWithSpaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Indent With &amp;Spaces</string>
   </property>
  </action>
  <action name="actionSelectFont">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/preferences-desktop-font.png</normaloff>:/images/16x16/preferences-desktop-font.png</iconset>
   </property>
   <property name="text">
    <string>Select &amp;Font...</string>
   </property>
  </action>
  <action name="actionFindReplace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/edit-find-replace.png</normaloff>:/images/16x16/edit-find-replace.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find/Replace</string>
   </property>
  </action>
  <action name="actionEvalManual">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/media-playback-start.png</normaloff>:/images/16x16/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Manual</string>
   </property>
   <property name="toolTip">
    <string>Manual Evaluation</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionEvalAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/media-skip-forward.png</normaloff>:/images/16x16/media-skip-forward.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Automatic</string>
   </property>
   <property name="toolTip">
    <string>Automatic Evaluation</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionEvalSteady">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/media-seek-forward.png</normaloff>:/images/16x16/media-seek-forward.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Steady</string>
   </property>
   <property name="toolTip">
    <string>Steady Evaluation</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionManageCustomActions">
   <property name="text">
    <string>&amp;Manage...</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionOnlineHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/help-faq.png</normaloff>:/images/16x16/help-faq.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Online Help</string>
   </property>
  </action>
  <action name="actionDarkTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Dark Theme</string>
   </property>
  </action>
  <action name="actionEvalReset">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/view-refresh.png</normaloff>:/images/16x16/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset Evaluation</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionFocusNextEditor">
   <property name="text">
    <string>Focus next editor</string>
   </property>
  </action>
  <action name="actionFocusPreviousEditor">
   <property name="text">
    <string>Focus previous editor</string>
   </property>
  </action>
  <action name="actionOpenContainingFolder">
   <property name="text">
    <string>Open containing folder</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="toolTip">
    <string>&amp;Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionFullScreenClose">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/16x16/window-close.png</normaloff>:/images/16x16/window-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
